tests/test_integration.py

```diff
@@ -152,38 +152,6 @@ def test_worker_single_trace_multiple_spans(self):
         eq_(payload[0][0]['name'], 'client.testing')
         eq_(payload[0][1]['name'], 'client.testing')
 
-    def test_worker_single_service(self):
```

Comment:
We should keep these and assert that calling `set_service_info` doesn't make an API call to `/v0.3/services`.